src/item/VanillaItems.php

```diff
@@ -626,6 +627,7 @@ private static function registerArmorItems() : void{
 		self::register("iron_chestplate", new Armor(new IID(Ids::IRON_CHESTPLATE), "Iron Chestplate", new ArmorTypeInfo(6, 241, ArmorInventory::SLOT_CHEST, material: ArmorMaterials::IRON()), [EnchantmentTags::CHESTPLATE]));
 		self::register("leather_tunic", new Armor(new IID(Ids::LEATHER_TUNIC), "Leather Tunic", new ArmorTypeInfo(3, 81, ArmorInventory::SLOT_CHEST, material: ArmorMaterials::LEATHER()), [EnchantmentTags::CHESTPLATE]));
 		self::register("netherite_chestplate", new Armor(new IID(Ids::NETHERITE_CHESTPLATE), "Netherite Chestplate", new ArmorTypeInfo(8, 593, ArmorInventory::SLOT_CHEST, 3, true, material: ArmorMaterials::NETHERITE()), [EnchantmentTags::CHESTPLATE]));
+		self::register("elytra", new Elytra(new IID(Ids::ELYTRA), "Elytra", new ArmorTypeInfo(0, 433, ArmorInventory::SLOT_CHEST, 0, false, null)));
```

Comment:
You can skip the optional parameters and Elutes must have its own class to prevent it from actually breaking by overriding it with an empty `onBroken()` method.